src/main/java/org/elasql/procedure/tpart/TPartStoredProcedureTask.java

```diff
@@ -7,29 +7,50 @@
 import org.elasql.schedule.tpart.sink.SunkPlan;
 import org.elasql.server.Elasql;
 import org.elasql.sql.PrimaryKey;
+import org.elasql.util.FeatureCollector;
+import org.elasql.util.TransactionFeaturesRecorder;
 import org.elasql.util.TransactionStatisticsRecorder;
 import org.vanilladb.core.remote.storedprocedure.SpResultSet;
+import org.vanilladb.core.server.VanillaDb;
+import org.vanilladb.core.server.task.Task;
 import org.vanilladb.core.util.Timer;
 
 public class TPartStoredProcedureTask
 		extends StoredProcedureTask<TPartStoredProcedure<?>> {
 	
+	private static class WaitingForStartingRecordTask extends Task {
+//		private static final int WARM_UP_TIME = 100_000;
+		private static final int WARM_UP_TIME = 0; // in milliseconds
+		@Override
+		public void run() {
+			try {
+				Thread.sleep(WARM_UP_TIME);
+			} catch (InterruptedException e) {
+				e.printStackTrace();
+			}
+			TransactionStatisticsRecorder.startRecording();
+			TransactionFeaturesRecorder.startRecording();
+		}
+	}
+	
 	static {
 		// For Debugging
 //		TimerStatistics.startReporting();
-		TransactionStatisticsRecorder.startRecording();
+//		TransactionStatisticsRecorder.startRecording();
+		VanillaDb.taskMgr().runTask(new WaitingForStartingRecordTask());
 	}
 
 	private TPartStoredProcedure<?> tsp;
 	private int clientId, connectionId, parId;
 	private long txNum;
-
+	private long txStartTime, sinkStartTime, sinkStopTime, threadInitStartTime;
+	private static long firstTxStartTime;
```

Comment:
Static variables should be put before all other variables separately. Please move this line to line 41~43. Like this:

```java
    VanillaDb.taskMgr().runTask(new WaitingForStartingRecordTask());
}

private static long firstTxStartTime;

private TPartStoredProcedure<?> tsp;
private int clientId, connectionId, parId;
```